changelog/fragments/reorder-docker-args.yaml

```diff
@@ -0,0 +1,31 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Support folks that may have `docker` aliased to `podman`, and Podman prefers options before positional arguments:
+
+      ```console
+      $ podman build --help | grep CONTEXT-DIRECTORY
+         podman build [command options] CONTEXT-DIRECTORY | URL
+      ```
+    kind: "change"
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: If you are using podman aliased as docker
+      body: >
+        Update the `Makefile`, change the argument of the `docker build` command from
+
+            docker build . -t ${IMG}
+
+        to
+             docker build -t ${IMG} .
+
+        Alternatively, you could replace it programmatically
+
+        ```console
+        $ sed -i 's/docker build . -t ${IMG}/docker build -t ${IMG} ./' $(git grep -l 'docker.*build \. ')
```

Comment:
For easy copying:
````suggestion
        ```sh
        sed -i 's/docker build . -t ${IMG}/docker build -t ${IMG} ./' $(git grep -l 'docker.*build \. ')
````